### PR TITLE
added solution to lc:577

### PR DESCRIPTION
**Pull request by Dhoni77**
✅ Updated readme
✅ Added solution file

### CODE REVIEW

File: solution/0500-0599/0577.Employee Bonus/README.md [hunk 1]
```diff
@@ -52,7 +52,14 @@ empId 是这张表单的主关键字
 ### **SQL**
 
 ```sql
-
+SELECT 
+    e.empid, b.bonus
```
Comment (yanglbme) [inline solution/0500-0599/0577.Employee Bonus/README.md:56]:
```suggestion
    e.name, b.bonus
```

Comment (yanglbme) [reply solution/0500-0599/0577.Employee Bonus/README.md:56]: > Write an SQL query to report the name and bonus amount of each employee with a bonus less than 1000.

**Review by yanglbme — CHANGES_REQUESTED** [review]

**Review by yanglbme — APPROVED** [review]
@Dhoni77 Thanks

**Comment by Dhoni77** [comment]
Done @yanglbme 👍

**Pull request merged**